### DISCUSSION TITLE
Why does typing in my domain take me to domain.com/login  ?

### DISCUSSION THREAD

**Discussion by mascalzone** [AzuraCast/AzuraCast#5788] [Q&A]
Hi I just installed azuracasta from doker. I had an old version and everything was fine. This instead if I try to connect to miaradio.com takes me to /miaradio.com/login.
You will understand that it is very annoying.
Does anyone know how to do? Why is this forced redirection done?

**Comment by BusterNeece** [comment]
@mascalzone By default, the main page always redirects to the login page if you're not logged in to AzuraCast. If you want the main page to redirect somewhere else for anonymous users, there's a setting in Custom Branding at the system level that lets you customize what URL it redirects to.

**Comment by mascalzone** [comment]
Yes. I found that function and now it works fine. I thank you..
